backend/src/services/udemy/udemy.service.ts

```diff
@@ -85,15 +87,23 @@ class Udemy {
       );
 
     while (fetchedModulesData.next) {
-      modules.push(...fetchedModulesData.results);
+      fetchedModulesData.results.map((item) => {
```

Comment:
As it's not actually about mapping but just iterating
```suggestion
      fetchedModulesData.results.forEach((item) => {
```